src/Groups/matrices/matrix_manipulation.jl

```diff
@@ -37,7 +37,7 @@ Return the `m x n` submatrix of `A` rooted at `(i,j)`
 """
 # TODO: eliminate this function again
 function submatrix(A::MatElem, i::Int, j::Int, nr::Int, nc::Int)
-   return A[i:i+nr-1, j:j+nr-1]
```

Comment:
Clearly this was simply a typo, one of the nr should have been nc. Can you adjust it that way, please?